Dockerfile.rhel7

```diff
@@ -1,8 +1,20 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.12 AS builder
+
+RUN dnf install -y git make go
```

Comment:
Why you're doing this, all of those tools are already part of the builder image.

Reply:
done